pkg/skaffold/deploy/docker/deploy_test.go

```diff
@@ -113,6 +119,44 @@ func TestDebugBindings(t *testing.T) {
 					},
 				},
 			},
+			portForwardResources: nil,
+		},
+		{
+			name: "two artifacts two bindings and two port forward resources",
```

Comment:
Initially, I added a test case to the TestDebugBindings function since it was the only function here. However, I did end up adding a separate function `TestFilterPortForwardingResources` to test specifically that. Let me know if I should revert the changes in this TestDebugBindings function.